pyproject.toml

```
@@ -15,7 +15,7 @@ include = [
 python = "^3.7"
 Flask = "^2.0.2"
 click = "^8.0.3"
-manim = "^0.13.1"
+manim = ">=0.13.1"
```

I've had huge problems with the `>=`  notation on manim. Shouldn't this work too?
```suggestion
manim = "^0"
```

---

For most practical reasons that works, but technically the section API PR was introduced relatively recently, so the plugin wouldn't work with older versions of manim. I chose to keep it at `v0.13.1` since that was the version where `manim_editor` was announced in discord.

Also, when manim hits a `1.0` release this would have to be updated again, which may or may not be necessary :shrug:

---

That's actually a very good point, so we can keep your idea. Could you run `poetry update` and then merge this?
I could do that too but this way the PR is attributed entirely to you.